website/docs/enterprise/getting-started/install-enterprise-cli.mdx

```diff
@@ -0,0 +1,138 @@
+
+### Verifying Entitlement
+
+Weave GitOps Enterprise Entitlement is your obtained license to use our product. The entitlement file is a kubernetes secret that contains your licence. The bootstrap CLI will check if the entitlement manifest exists on the management cluster, and it will check if it has valid content and the entitlement is not expired. 
+To get the entitlement secret please contact *sales@weave.works*, then apply it on your management cluster with the name `weave-gitops-enterprise-credentials` under `flux-system` namespace. 
+
+### Verifying flux
+
+Weave GitOps Enterprise runs on top of flux, the bootstrap CLI will check if flux is installed on the management cluster, and it will verify that it has the right version with valid git repository setup, and it is able to reconcile flux components properly. 
+If flux is installed, but doesn't have a valid installation, the bootstrap CLI will terminate pending the fix or uninstall of current flux installation.
+If flux is not installed at all, the bootstrap CLI offers the option to bootstrap flux. 
+
+### Adding private key path and private key password
+
+Private key will be used to push WGE resources to the cluster's git repository.
+Priave key password is not optional if the private key is encrypted with it
+
+### Selecting WGE version
+
+The bootstrap CLI will prompt you to choose from the latest 3 versions of Weave GitOps Enterprise.
+
+### Create admin credentials
+
+Creates admin secret with username and password to be able to access the dashboard. If you already have previous admin credentials on your cluster, the installation will prompt you if you want to continue with the old credentials or exit and revoke them and re-run the installation.
+
+### Select the domain
+
+choose between 2 methods to access the dashboard either locally or via external domain
+Localhost can be useful in air-gapped environments or testing on local-host
+External Domain will provide you with a domain to access your dashboard UI.
+It will need to be configured by your service provider for example in AWS you need to configure your route 53: https://aws.amazon.com/route53/
+
+{/* TODO: @waleedhammam to add more information about external dns */}
+
+### Configure OIDC
+
+OIDC configuration will enable you to login with OIDC provider beside admin credentials.
+Add your OIDC `DiscoveryUrl` this will verify that OIDC is accessible and get the issuerUrl from the OIDC settings.
+Then add the `clientID` & `clientSecret` that you have configured on your OIDC static-clients.
+Note: don't forget to add a new static-client on your OIDC provider settings with the redirectURI `your-domain/oauth2/callback` for example `http://localhost:3000/oauth2/callback`
+Also you can configure the OIDC later by running the following command `bootstrap auth --type=oidc`
+
+### Access the dashboard
+
+Via the link from the installation success message
+
+## Appendix
+
+### CLI configurations
```

Comment:
how do we expect to maintain/update the documentation, if there is an overhead or manual, it might get quickly outdated